src/main/scala/io/iohk/praos/domain/Genesis.scala

```diff
@@ -0,0 +1,12 @@
+package io.iohk.praos.domain
+
+import io.iohk.praos.crypto.Seed
+
+
+/**
+  * A genesis block, used at the very start of the blockchain and at the start of each epoch.
+  *
+  * @param genesisDistribution The stake distribution to be used.
+  * @param genesisNonce        The nonce to be used.
+  */
+case class Genesis(genesisDistribution: StakeDistribution, genesisNonce: Seed)
```

Comment:
Should we add a comment regarding what `Genesis` means in this context, and that it is used in every slot?

Reply:
ok!